uxkit-screenshare/src/main/java/com/voxeet/uxkit/screenshare/ScreenShareMediaPresentationProvider.java

```diff
@@ -0,0 +1,53 @@
+package com.voxeet.uxkit.screenshare;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.voxeet.uxkit.common.logging.ShortLogger;
+import com.voxeet.uxkit.common.UXKitLogger;
+import com.voxeet.uxkit.presentation.provider.AbstractMediaPlayerProvider;
+
+/**
+ * Manage and create screenshare videos
+ */
+public class ScreenShareMediaPresentationProvider extends AbstractMediaPlayerProvider<ScreenShareMediaPresentationView> {
```

Comment:
We should use Kotlin for new modules

Reply:
Kotlin will be part of the next major update. For now it's about coherence (moreover Kotlin doesn't bring any value here)